src/common/localization/en-us.ts

```diff
@@ -143,6 +143,8 @@ export const english: IAppStrings = {
         settings: "Connection Settings",
         instructions: "Please select a connection to edit",
         save: "Save Connection",
+        corsWarning: "Warning: when using VoTT in the browser, assets from this provider may not load correctly \
```

Comment:
If someone has better copy here, I'm all ears!

Reply:
This issue is with exporting vs. loading.  Maybe something to the effect of "Warning: When using VoTT in the browser some assets from this provider may not export correctly due to CORS (cross origin resource sharing) restrictions.

Reply:
got it - I thought it was mainly around being able to display at all. I'll update to this.  One more question: Should we do something similar for blob, or just call it good? I'm leaning towards the latter at this point...

Reply:
We have something in the help docs about enabling CORS in blob, but it would be better if there was a message inline within the app since that would limit actual loading of images.

Reply:
Since I went back to fix up the translation, I went ahead and did this for blob as well.